packages/catalog-process/src/services/catalogService.ts

```diff
@@ -438,6 +438,9 @@ export function catalogServiceBuilder(
         riskAnalysis: [],
         isSignalHubEnabled: seed.isSignalHubEnabled,
         isDelegable: seed.isDelegable,
+        isClientAccessDelegable: seed.isDelegable
+          ? seed.isClientAccessDelegable
+          : undefined,
```

Comment:
Just to make the data more coherent I prefer this condition even if it's more complex. 

```suggestion
        isClientAccessDelegable: match(seed.isDelegable)
          .with(P.nullish, () => undefined)
          .with(false, () => false)
          .with(true, () => seed.isClientAccessDelegable)
          .exhaustive(),
```